src/components/Person/Person.jsx

```diff
@@ -1 +1,54 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+  const partnerGender = sex === 'f'
+    ? 'husband'
+    : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {
+          isMarried
+            ? `${partnerName} is my ${partnerGender}`
+            : `I am not married`
+        }
+      </p>
+    </section>
+  );
+};
+
+Person.defaultProps = {
+  person: {
+    age: null,
+    partnerName: null,
+  },
+};
+
+Person.propTypes = {
+  person: PropTypes.shape({
+    name: PropTypes.string.isRequired,
+    age: PropTypes.number,
+    sex: PropTypes.symbol.isRequired,
```

Comment:
maybe it's better to use: sex: PropTypes.oneOf(['f', 'm']).isRequired,

Reply:
agree let's use `oneOf` for `sex`